manifest.yml

```diff
@@ -0,0 +1,11 @@
+manifest_version: 1
+name: ImageTorchEncoder
+kind: encoder
```

Comment:
I think you need to add an alias here, this is importatnt for hubble in some way:
```
alias: ImageTorchEncoder
```